Files/Views/Pages/Properties.xaml

```diff
@@ -99,7 +99,7 @@
                 HorizontalAlignment="Right"
                 Click="OKButton_Click"
                 Content="OK"
-                Style="{ThemeResource ButtonRevealStyle}" />
+                Style="{ThemeResource AccentButtonStyle}" />
```

Comment:
You can use `{StaticResource AccentButtonStyle}` here.

Reply:
ok

Reply:
In that case `AccentButton.xaml` and `packages.config` no more needed.

Reply:
removed